docs/developer/export-products.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: Exporting Products
+sidebar_label: Exporting products
```

Comment:
This is only required if the sidebar label (what appears in the sidebar) is different from the title (what appears at the top of the page).